vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/SpreadsheetFilterTable.java

```diff
@@ -300,21 +296,4 @@ public void unRegisterFilter(PopupButton popupButton,
             popupButtonToFiltersMap.remove(popupButton);
         }
     }
-
-    @Override
-    public void onComponentEvent(ComponentEvent event) {
-        if (event instanceof ClickEvent)
-            event.getSource().getParent().ifPresent(parent -> {
-                if (parent instanceof PopupButton
-                        && popupButtonToFiltersMap.containsKey(parent)) {
-                    HashSet<SpreadsheetFilter> filters = popupButtonToFiltersMap
-                            .get(parent);
-                    for (SpreadsheetFilter filter : filters) {
-                        filter.clearFilter();
-                    }
-                    ((PopupButton) parent).markActive(false);
-                    onFiltersUpdated();
-                }
-            });
-    }
```

Comment:
Since [this change](https://github.com/vaadin/spreadsheet/commit/768f32b0defebf3c3641c3ca71b5db3dd5ecf803#diff-e4a143b61e9821e31396ed71e6e54e2cf9a7fc5974ccfb3ca9c07673cc46e3a6R213), the "clear filters" button has actually cleared all the filters in the table, not just the filters associated with the specific popup button.